### DISCUSSION TITLE
How can i update the logs who reveal the password?

### DISCUSSION THREAD

**Discussion by zeeshantariq08** [filamentphp/filament#12206] [Help]
### Package

Form builder

### Package Version

v3.0.0

### How can we help you?

@leandrocfe do you know?
```
Forms\Components\TextInput::make('password')
                                ->password()
                                ->revealable()
```

**Comment by leandrocfe** [comment]
Sorry, what does it mean?

**Reply by zeeshantariq08** [reply]
i am trying to update the logs, who reveal the password , how can I add action when a eye icon click
